packages/backend/src/controllers/authentication.ts

```diff
@@ -161,10 +179,18 @@ export const isAuthenticated: RequestHandler = (req, res, next) => {
         next(new AuthorizationError(`Failed to authorize user`));
     }
 };
+
 export const unauthorisedInDemo: RequestHandler = (req, res, next) => {
     if (lightdashConfig.mode === LightdashMode.DEMO) {
         throw new AuthorizationError('Action not available in demo');
     } else {
         next();
     }
 };
+
+// export const allowApiKeyAuthentication: RequestHandler = (req, res, next) => {
+//     if (req.isAuthenticated()) {
+//         next();
+//     }
+//     passport.authenticate('localapikey', { session: false })(req, res, next);
+// };
```

Comment:
@owlas don't we need to use this middleware in the routes that can be used by the api token ? otherwise I don't understand where we are authenticating the user with the token.